src/app/core/menu/menu.component.scss

```diff
@@ -13,7 +13,7 @@ $primary-color: #211056;
   font-weight: 600;
   z-index: 10;
   .title {
-    font-size: 1.2rem;
+    font-size: 25px;
```

Comment:
Why change to px?

Reply:
Because we needed it a little bit bigger, and 25px looks big enough in both desktop and mobile (more in mobile, as wanted), whereas using rem units, the big enough for mobile is too big for desktop and to ake it work we would have to use a media query, wich is much more work than just using px